holmes/plugins/toolsets/datadog/toolset_datadog_metrics.py

```
@@ -0,0 +1,531 @@
+import os
+from typing import Any, Optional, Dict, Tuple
+from holmes.core.tools import (
+    CallablePrerequisite,
+    StructuredToolResult,
+    Tool,
+    ToolParameter,
+    ToolResultStatus,
+    Toolset,
+    ToolsetTag,
+)
+from pydantic import AnyUrl, BaseModel
```

Ruff (0.12.2)

`pydantic.AnyUrl` imported but unused

Remove unused import

(F401)